src/widgets/AutocompleteSelector.js

```diff
@@ -100,7 +100,7 @@ export class AutocompleteSelector extends React.PureComponent {
           placeholder={placeholderText}
           onChangeText={this.onChangeText}
           style={[localStyles.text, localStyles.searchBar]}
-          color="white"
+          WHITE="#FFFFFF"
```

Comment:
There is no prop `WHITE` on `SearchBar` this doesn't make any sense

Reply:
check on index.js. Value assigned to WHITE IN index.js.  Imported WHITE to searchBar then use it in Autocompleteselector. That's how it works.

Reply:
Josh's comment here is correct.

`SearchBar` is a component used by `AutocompleteSelector` (this file that we're commenting on). `WHITE` was not a prop previously, it was `color` and `placeholderTextColor`. You have edited SearchBar, but not in anyway that enables what you're doing here, not that it is necessary. 

All that is required for this PR is to change the imported SearchBar to be the implementation within this repo (as you have done), and to change value provided to `color` and `placeholderTextColor` to be the constant `WHITE` that you added to the exports in globalStyles.

Reply:
I misunderstood the changes that you requested me to do and went on wrong direction instead.
so next time I better ask for clarification on comments to save time.
There are some changes that Chris suggest is added as below
<img width="520" alt="Screen Shot 2019-12-16 at 8 58 53 PM" src="https://user-images.githubusercontent.com/57238075/70889335-3ad49a80-2047-11ea-9beb-58ca3a5bdefc.png">